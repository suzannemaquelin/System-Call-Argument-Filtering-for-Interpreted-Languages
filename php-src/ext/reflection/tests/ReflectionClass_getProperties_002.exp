Test invalid arguments:
Warning: ReflectionClass::getProperties() expects parameter 1 to be integer, string given in %s on line 4

Warning: ReflectionClass::getProperties() expects at most 1 parameter, 2 given in %s on line 5