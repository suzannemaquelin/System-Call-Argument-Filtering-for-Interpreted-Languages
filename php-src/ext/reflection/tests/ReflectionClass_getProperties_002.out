Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory

Test invalid arguments:
Warning: ReflectionClass::getProperties() expects parameter 1 to be integer, string given in /root/php-src/ext/reflection/tests/ReflectionClass_getProperties_002.php on line 4

Warning: ReflectionClass::getProperties() expects at most 1 parameter, 2 given in /root/php-src/ext/reflection/tests/ReflectionClass_getProperties_002.php on line 5